docs/_docs/extensions-and-integrations/spring/spring-tx.adoc

```diff
@@ -0,0 +1,471 @@
+       xsi:schemaLocation="
+           http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+           http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx.xsd">
+    <-- Provide configuration bean. -->
+    <bean id="transactionManager" class="org.apache.ignite.transactions.spring.SpringTransactionManager">
+        <property name="configuration">
+            <bean id="gridCfg" class="org.apache.ignite.configuration.IgniteConfiguration">
+                ...
+            </bean>
+        </property>
+    </bean>
+
+    <-- Use annotation-driven transaction configuration. -->
+    <tx:annotation-driven/>
+</beans>
+----
+--
+
+[discrete]
+=== 3. Specifying path to Apache Ignite XML node configuration file
+
+In this case Apache Ignite node instance will be started automatically by the manager based on provided configuration.
+
+[tabs]
+--
+tab:Java[]
+[source,java]
+----
+@Configuration
+@EnableTransactionManagement
+public class SpringApplicationConfiguration {
+    @Bean
+    public SpringTransactionManager transactionManager() {
+        SpringTransactionManager mgr = new SpringTransactionManager();
+
+        mgr.setConfigurationPath("<path to an Apache Ignite configuration XML file (path can be absolute or relative to `IGNITE_HOME`)>")
+
+        return mgr;
+    }
+}
+----
+tab:XML[]
+[source,xml]
+----
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:tx="http://www.springframework.org/schema/tx"
+       xsi:schemaLocation="
+           http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+           http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx.xsd">
+    <-- Provide configuration file path. -->
+    <bean id="transactionManager" class="org.apache.ignite.transactions.spring.SpringTransactionManager">
+        <property
+            name="configurationPath"
+            value="<path to an Apache Ignite configuration XML file (path can be absolute or relative to `IGNITE_HOME`)"/>
+    </bean>
+
+    <-- Use annotation-driven transaction configuration. -->
+    <tx:annotation-driven/>
+</beans>
+----
+--
+
+[NOTE]
+====
+If no connection configuration is specified, `SpringTransactionManager` will use Apache Ignite node instance with the
+default name. If it does not exist, an exception will be thrown during the `SpringTransactionManager` startup.
+====
+
+[IMPORTANT]
+====
+[discrete]
+No matter what configuration approach you chose, the same Apache Ignite node instance must be used to both initialize the
+transaction manager and perform transactional operations. In case Apache Ignite node was started by the transaction manager,
+you can obtain its instance in your application through `Ignition.ignite("<name of the Apache Ignite node instance>");`.
```

Comment:
```suggestion
Regardless the configuration approach you chose, the same Apache Ignite node instance must be used to initialize the transaction manager and perform the transactional operations. In case Apache Ignite node was started by the transaction manager, you can obtain its instance in your application through `Ignition.ignite("<name of the Apache Ignite node instance>");`.
```